packages/nextjs/pages/blockexplorer/transaction/[txHash].tsx

```diff
@@ -94,14 +94,16 @@ const TransactionPage: NextPage = () => {
                   <strong>Function called:</strong>
                 </td>
                 <td>
-                  {functionCalled === "0x" ? (
-                    "This transaction did not call any function."
-                  ) : (
-                    <>
-                      <span className="mr-2">{getFunctionDetails(transaction)}</span>
-                      <span className="badge badge-primary font-bold">{functionCalled}</span>
-                    </>
-                  )}
+                  <div className="w-full md:max-w-[600px] lg:max-w-[800px] overflow-x-auto whitespace-nowrap scrollbar-thin scrollbar-thumb-primary scrollbar-track-secondary">
```

Comment:
what about sm/xs?

Reply:
The table in [txHash].tsx page overflows the containers when the viewport width is below 600px. I plan to fix it in another PR since this one introduces a package and changes the page already. 
But maybe we can add it as a commit. Thanks for pointing this out!